mobilizon_bots/storage/query.py

```diff
@@ -26,6 +31,28 @@ async def prefetch_event_relations(queryset: QuerySet[Event]) -> list[Event]:
     )
 
 
+@atomic(CONNECTION_NAME)
+async def publications_with_status(
+    status: PublicationStatus,
+    event_mobilizon_id: Optional[UUID] = None,
+    from_date: Optional[Arrow] = None,
+    to_date: Optional[Arrow] = None,
+) -> Iterable[Publication]:
+    query = Publication.filter(status=status)
+
+    if event_mobilizon_id:
+        query = query.prefetch_related("event").filter(
+            event__mobilizon_id=event_mobilizon_id
+        )
+
+    if from_date:
```

Comment:
use the refactored function added in the previous PR

Reply:
Yes, now it's fixed